integration-tests/test/lib/JavabuilderConnectionHelper.js

```diff
@@ -64,37 +53,49 @@ class JavabuilderConnectionHelper {
     socket.onclose = onClose;
     socket.onerror = onError;
   }
+}
 
-  generateToken(miniAppType, sessionId) {
-    const issuedAtTime = Date.now() / 1000 - 3;
-    const expirationTime = issuedAtTime + 63;
-    const payload = {
-      iat: issuedAtTime,
-      iss: "integration-tests",
-      exp: expirationTime,
-      uid: this.getRandomId(),
-      level_id: "none",
-      execution_type: "RUN",
-      mini_app_type: miniAppType,
-      verified_teachers: this.getRandomId(),
-      options: "{}",
-      sid: sessionId,
-      can_access_dashboard_assets: false
-    };
+const getRandomId = () => uuidv4();
 
-    const key = crypto.createPrivateKey({
-      key: PRIVATE_KEY,
-      passphrase: PASSWORD,
-    });
+function generateToken(miniAppType, sessionId) {
```

Comment:
Why this change? just curious

Reply:
Ah this is left over from a previous WIP iteration, I'll move back.